src/geometries/multigeom.jl

```diff
@@ -48,12 +48,14 @@ Base.isapprox(m₁::Multi, m₂::Multi; atol=atol(lentype(m₁)), kwargs...) =
 # POLYTOPE
 # ---------
 
-vertex(m::MultiPolytope, ind) = vertices(m)[ind]
+vertex(m::MultiPolytope, ind) = getindex(Base.iterate(Base.Iterators.drop(eachvertex(m), ind - 1)), 1) # nth(itr, n)
```

Comment:
```suggestion
vertex(m::MultiPolytope, ind) = first(iterate(Iterators.drop(eachvertex(m), ind - 1)))
```

Reply:
if using `first` the `iterate` is not needed!
can just be `first(Iterators.drop(eachvertex(m), ind-1))`

Reply:
that is even better!